src/mon/MonmapMonitor.cc

```diff
@@ -243,8 +243,9 @@ bool MonmapMonitor::preprocess_command(MonOpRequestRef op)
 
   if (prefix == "mon stat") {
     mon->monmap->print_summary(ss);
-    ss << ", election epoch " << mon->get_epoch() << ", quorum " << mon->get_quorum()
-       << " " << mon->get_quorum_names();
+    ss << ", election epoch " << mon->get_epoch() << ", leader "
+       << mon->get_leader() << " " << mon->get_leader_name()
```

Comment:
I think it would be nice to return the proper monitor name here, instead of just the monitor's id. I think we want to make sure we get a `mon.<id>` here instead of just `<id>`.